src/Sylius/Bundle/LocaleBundle/Resources/config/services.xml

```diff
@@ -42,7 +42,7 @@
         <service id="Sylius\Bundle\LocaleBundle\Context\RequestHeaderBasedLocaleContext">
             <argument type="service" id="request_stack" />
             <argument type="service" id="sylius.locale_provider" />
-            <tag name="sylius.context.locale" priority="256" />
+            <tag name="sylius.context.locale" priority="32" />
```

Comment:
This change seems to be irrelevant to the main purpose of this PR. Is it related to: https://github.com/Sylius/Sylius/pull/13708#issuecomment-1095396785?